test/tstMinimumSpanningTreeGoldenTest.cpp

```diff
@@ -52,8 +52,8 @@ auto parsePointsFromCSVFile(std::string const &filename)
     Tokenizer tok(line);
     auto first = tok.begin();
     auto const last = tok.end();
-    points.emplace_back(std::stof(*first++), std::stof(*first++),
-                        std::stof(*first++));
+    points.push_back(
```

Comment:
```suggestion
    points.push_back(  // NOLINT(modernize-use-emplace)
```

Reply:
(haven't tested whether you need it there or after the semicolon but that will shot clang-tidy up)